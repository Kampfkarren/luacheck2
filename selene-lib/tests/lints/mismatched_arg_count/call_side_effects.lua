local function foo(a, b, c)
end

foo(a, call())
foo(a, call(), c)
foo(a, b, call())
foo(a, b, c, call())
