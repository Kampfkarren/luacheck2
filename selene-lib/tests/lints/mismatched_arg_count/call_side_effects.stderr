error[mismatched_arg_count]: this function takes 3 arguments but atleast 4 arguments were supplied
  ┌─ call_side_effects.lua:1:16
  │
1 │ local function foo(a, b, c)
  │                --- note: function defined here
  ·
7 │ foo(a, b, c, call())
  │ ^^^^^^^^^^^^^^^^^^^^ expected 3 arguments

